site/src/sphinx/advanced-spring-webflux-integration.rst

```diff
@@ -0,0 +1,80 @@
+.. _`Spring Boot Reference Guide`: https://docs.spring.io/spring-boot/docs/current/reference/html/
+.. _`armeria-examples`: https://github.com/line/armeria-examples
+
+.. _advanced-spring-webflux-integration:
+
+Using Armeria as a web server of a reactive Spring application
+==============================================================
+
+A user can use Armeria as a web server of his or her reactive Spring application by adding
+`spring-boot-webflux-starter` to the dependencies.
+
+For Maven:
+
+.. code-block:: xml
+
+    <dependency>
+        <groupId>com.linecorp.armeria</groupId>
+        <artifactId>spring-boot-webflux-starter</artifactId>
+    </dependency>
+
+For Gradle:
+
+.. code-block:: groovy
+
+    dependencies {
+        compile 'com.linecorp.armeria:spring-boot-webflux-starter'
+    }
+
+It automatically configures the Armeria web server by referring to `application.yml` when the application starts up.
+A user can configure the server with the properties provided by Spring framework, because it replaces
+the existing web server, such as Tomcat or Netty, with Armeria web server. The following is
+a simple example for configuring the server:
+
+.. code-block:: yml
+
+    server:
+      address: 127.0.0.1
+      port: 8080
+
+For a user who wants to customize the web server, :api:`ArmeriaServerConfigurator` is provided.
+The user can customize the server by defining a bean of the type in the configuration as follows:
+
+.. code-block:: java
+
+    @Configuration
+    public class ArmeriaConfiguration {
+        /**
+         * A user can configure the server by providing an ArmeriaServerConfigurator bean.
+         */
+        @Bean
+        public ArmeriaServerConfigurator armeriaServerConfigurator() {
+            return builder -> {
+                // Customize the server using the given ServerBuilder.
```

Comment:
How about adding some example builder calls?

Reply:
I added an example builder calls in the `example`: ...g-boot-webflux/src/main/java/example/springframework/boot/webflux/HelloConfiguration.java

Is it okay to copy that to here? What do you think?

Reply:
Yes, let's make a copy here.. Can't think of a better way yet.